packages/manager/modules/dbaas-logs/src/logs/detail/streams/subscriptions/subscriptions.component.js

```diff
@@ -0,0 +1,8 @@
+import controller from './streams-subscriptions.controller';
+import template from './streams-subscriptions.html';
+
+export default {
+  controller,
+  controllerAs: 'ctrl',
```

Comment:
**nitpick:** You can ignore this and use the default value of `$ctrl`
```suggestion
```

Reply:
Removed, also switched to `$ctrl` in the related HTML file